Articles/Blog/2024-07-BSSwFellows2023.md

```diff
@@ -0,0 +1,208 @@
+# 2023 BSSw Fellows: Projects and Perspectives
+
+Read about the 2023 BSSw Fellows and their contributions to the BSSw community!
+
+<!-- TODO: Get new image -->
```

Comment:
@gonsie are you really planning to get a new image?  Or is this a carry over from the past?

If a new image is coming, when do we expect to receive it?